src/main/java/edu/harvard/iq/dataverse/api/BatchServiceBean.java

```diff
@@ -95,6 +98,8 @@ public JsonArrayBuilder handleDirectory(DataverseRequest dataverseRequest, File
             }
         }
         for (File file : dir.listFiles()) {
+		if(null!=file)
```

Comment:
huh?

Reply:
@oscardssmith could you clarify which indexing?

Reply:
oops, indenting